appendices/VK_NV_compute_shader_derivatives.txt

```diff
@@ -24,12 +24,12 @@ compute shaders to use built-ins that evaluate compute derivatives
 explicitly or implicitly.
 Derivatives will be computed via differencing over a 2x2 group of shader
 invocations.
-The code:DerivativeGroupQuadsNV execution mode assembles shader invocations
-into 2x2 groups, where each group has x and y coordinates of the local
-invocation ID of the form (2m+{0,1}, 2n+{0,1}).
-The code:DerivativeGroupLinearNV execution mode assembles shader invocations
-into 2x2 groups, where each group has local invocation index values of the
-form 4m+{0,1,2,3}.
+The code:ComputeDerivativeGroupQuadsNV execution mode assembles shader
+invocations into 2x2 groups, where each group has x and y coordinates of the
+local invocation ID of the form (2m+{0,1}, 2n+{0,1}).
+The code:ComputeDerivativeGroupLinearNV execution mode assembles shader
+invocations into 2x2 groups, where each group has local invocation index
+values of the form 4m+{0,1,2,3}.
```

Comment:
Isn't the execution mode name correct without `Compute` (and it's the capability that has `Compute` on it?

Reply:
Yea, I think so. I was fixing a link to a derivatives chapter that does not exist anymore. I don't even remember changing it. Damn heat...

Reply:
should be gone now